src/utils/general_look_at_data.py

```diff
@@ -70,12 +70,9 @@ def display_direct_missing(df):
 def display_indirect_missing(df):
     # Filter rows with missing percentage greater than 0
     filtered_stats = df[df.drop(columns='feature_type').gt(0).any(axis=1)]
-    
-    # Get indirect missing columns with at least one feature above 0.0
-    indirect_missing_cols = filtered_stats.drop(columns=['missing_count', 'missing_percentage'])
-    indirect_missing_cols = indirect_missing_cols.columns[indirect_missing_cols.any()]
+
     # Return indirect missing columns
-    return filtered_stats[indirect_missing_cols]
+    return filtered_stats.drop(columns=['missing_count', 'missing_percentage'])
```

Comment:
In this case, you will return all features regardless of the percentage of missing... previous code returns the columns that have at least one feature above 0.0...